tests/conftest.py

```diff
@@ -734,7 +735,6 @@ def tm():
     # Create a new transaction manager for dependant test cases
     tm = transaction.TransactionManager(explicit=True)
     tm.begin()
-    tm.doom()
```

Comment:
Flagging this: I had to remove this otherwise the task would refuse to commit the transaction mid-task. This doesn't seem to affect any other tests and I _think_ the database will be reset anyways, so there's no harm here. This was added in #16031.